guardian/explain-maker
# Long explainer checkbox

This introduces the 'Long explainer' checkbox allowing users to specify if an explainer is
Expandable or Flat. Note that this currently doesn't affect the workings of the Editor itself,
but the information is recorded.

Testing:
- run the code
- click on the checkbox
- reload the page. The checkbox should be in "checked" state.
- click on the checkbox
- reload the page. The checkbox should be in "un-checked" state.
